src/xml/error.cr

```diff
@@ -11,22 +11,9 @@ class XML::Error < Exception
     super(message)
   end
 
-  @@errors = [] of self
-
-  # :nodoc:
-  protected def self.add_errors(errors)
-    @@errors.concat(errors)
-  end
-
   @[Deprecated("This class accessor is deprecated. XML errors are accessible directly in the respective context via `XML::Reader#errors` and `XML::Node#errors`.")]
```

Comment:
The deprecation is redundant if it raises at compilation, isn't it? Will the warning be printed still? Or would it be better to incorporate the pointer to `XML::Reader#errors` into the raised error?

Reply:
No, they serve different purposes. The compiler won't print the warning, only the error. The annotation is still useful for documentation, for example.

Reply:
OK, but the error should mention the alternative.